dask_geomodeling/raster/spatial.py

```diff
@@ -102,14 +102,20 @@ def expand_request_meters(request, radius_m=1):
 
 class Dilate(BaseSingle):
     """
-    Perform binary dilation on specific values. Dilation is done in the
-    order of the values parameter.
-
-    :param store: raster to perform dilation on
-    :param values: only dilate pixels that have these values
-
-    :type store: RasterBlock
-    :type values: list
+    Perform spatial dilation on specific cell values.
+
+    Cells with values in the supplied list are spatially dilated by one cell in each direction (including diagonals). 
+    Dilation is performed in the order of the values parameter.
+    
+    Args:
+      store (RasterBlock): Raster to perform dilation on
+      values (list): Only cells with these values are dilated
+
+    Returns:
+      Rasterblock where cells in values list are dilated
+    
+    See also:
+      https://homepages.inf.ed.ac.uk/rbf/HIPR2/dilate.htm
```

Comment:
Don't link to sites like this, this like has a high chance to become unavailable.
Wiki is better: https://en.wikipedia.org/wiki/Dilation_%28morphology%29